flexget/plugins/filter/series.py

```diff
@@ -1451,6 +1506,22 @@ def on_task_start(self, task, config):
                     except ValueError as e:
                         raise plugin.PluginError(e)
 
+def _add_alt_name(alt, db_series, series_name, session):
+    alt = unicode(alt)
+    db_series_alt = session.query(AlternateNames).join(Series).filter(AlternateNames.alt_name == alt).first()
```

Comment:
Don't think we need to join to Series table here, since we aren't filtering on anything from there.